.github/workflows/build_icons.yml

```diff
@@ -44,15 +44,15 @@ jobs:
         if: success() 
         run: npm run build-css
 
-      - name: Upload screenshot of the newly made icons
-        id: imgur_step
-        uses: devicons/public-upload-to-imgur@v2.2.2
-        if: success()
-        with:
-          # will have "new_icons.png" and "new_svgs.png"
-          # in that order (cause sorted alphabetically)
-          path: ./screenshots/*.png
-          client_id: ${{secrets.IMGUR_CLIENT_ID}}
+      # - name: Upload screenshot of the newly made icons
+      #   id: imgur_step
+      #   uses: devicons/public-upload-to-imgur@v2.2.2
+      #   if: success()
+      #   with:
+      #     # will have "new_icons.png" and "new_svgs.png"
+      #     # in that order (cause sorted alphabetically)
+      #     path: ./screenshots/*.png
+      #     client_id: ${{secrets.IMGUR_CLIENT_ID}}
```

Comment:
This part is skipped by comment, but it can be removed.

Reply:
Let's keep it as-is so that we can fix it in the future. 👍